views/partials/gamer.ejs

```diff
@@ -13,6 +13,13 @@
 
     <div class="card-body demo-supporting-text mdl-card__supporting-text details-container">
 
+
+        <% if (gamer.num_site_hits > 0) { %>
+            <small>
+                <%= gamer.num_site_hits %>   <%= gamer.num_site_hits == 1 ? 'view' : 'views' %>
```

Comment:
What do you think about nixing logic stuff here and doing it all on the server when we sanitize the gamer anyway? meaning something like

gamer.pretty_views = `${gamer.num_site_hits} ${gamer.num_site_hits === 1 ? 'view' : 'views'}`;

`<%= gamer.pretty_views %>`